openslides/agenda/models.py

```diff
@@ -372,14 +380,14 @@ class Speaker(RESTModelMixin, models.Model):
 
     objects = SpeakerManager()
 
-    user = models.ForeignKey(User)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL)
```

Comment:
`settings.AUTH_USER_MODEL` is always a string but this is valid, see example in the box [here](https://docs.djangoproject.com/en/1.8/topics/auth/customizing/#referencing-the-user-model).